backend/subroker/urls.py

```diff
@@ -16,6 +16,7 @@
 
     #Content
     path('content/', views.content_list),
+    path('content/search/<str:search_str>/', views.content_search),
```

Comment:
@muhwagwa how about using query string in views.content_list instead of using a new URL path?

Reply:
You mean **'content/<str:search_str>'** right? We had some problem because we already have **path('content/<int:content_id>/', views.content_detail)** so I changed it that way! But if you meant something else, please tell me!

Reply:
@muhwagwa Oh i mean like this **'content/?name=avatar'** then we can integrate search with content_list view function.

Reply:
@arclic I see! I'll create an issue and work on it